addons/udes_stock/models/stock_quant.py

```diff
@@ -49,12 +49,15 @@ def get_quantity(self):
         """Returns the total quantity of the quants in self"""
         return sum(self.mapped("quantity"))
 
-    def get_quantities_by_key(self, get_key=lambda q: q.product_id, only_available=False):
+    def get_quantities_by_key(
+            self, get_key=lambda q: q.product_id, only_available=False, product_quantity=None
+    ):
         """Returns a dictionary with the total quantity per product, mapped by product_id.
         :kwargs:
             - only_available: Boolean
                 Whether to include those reserved or not in the grouping
             - get_key: a callable which takes a quant and returns the key
+            - product_quantity: Float Needed not to get all available products
```

Comment:
Needed not? Not sure I follow

Reply:
Not needed at least for now , because needed information is on products_info

Reply:
Can you explain the changes here? I'm not sure I follow :/ Maybe we can have a call.